androidwalletdemo/README.md

```diff
@@ -9,4 +9,9 @@ various examples:
  - [Generating keys and initializing a wallet - WalletRepository::initializeWallet](./src/main/java/com/lightspark/androidwalletdemo/wallet/WalletRepository.kt)
  - [Unlocking a wallet to make payments - WalletRepository::attemptKeyStoreUnlock](./src/main/java/com/lightspark/androidwalletdemo/wallet/WalletRepository.kt)
  - [Creating, Decoding, and Paying Invoices - PaymentRepository.kt](./src/main/java/com/lightspark/androidwalletdemo/wallet/PaymentRepository.kt)
- - [Scanning an invoice QR code - InvoiceQrScanner.kt](./src/main/java/com/lightspark/androidwalletdemo/sendpayment/InvoiceQrScanner.kt)
\ No newline at end of file
+ - [Scanning an invoice QR code - InvoiceQrScanner.kt](./src/main/java/com/lightspark/androidwalletdemo/sendpayment/InvoiceQrScanner.kt)
+
+### How to run
+
+1. Set up the [demo JWT server](https://github.com/lightsparkdev/js-sdk/tree/main/apps/examples/jwt-server)
+2. Add a `gradle.properties` file in this directory and add the ngrok URL as `jwtServerUrl`.
```

Comment:
hm i've read in a couple places local.properties shouldn't be manually modified since it's automatically generated by android studio. But i'm down to do this if you still think it's fine

Reply:
Interesting haha, it's definitely not fully auto-generated. I've used custom props in there in most big android projects I've worked on in the past. However, you can also pull out another properties file though since gradle.properties is usually committed to git, and in this case we don't want something committed. https://developer.android.com/build/gradle-tips#remove-private-signing-information-from-your-project